core/src/test/java/org/elasticsearch/index/mapper/MultiFieldTests.java

```diff
@@ -155,8 +156,9 @@ public void testBuildThenParse() throws Exception {
     // can to unnecessary re-syncing of the mappings between the local instance and cluster state
     public void testMultiFieldsInConsistentOrder() throws Exception {
         String[] multiFieldNames = new String[randomIntBetween(2, 10)];
+        HashSet<String> seenFields = new HashSet<>();
```

Comment:
Let's type the left-hand side as `Set<String>`.

Reply:
Ah, I prefer to have a concrete type for local variables. I will push the change.

Reply:
I think the less-specific types are better when you can get away with them so that you do not come to rely on implementation details of the concrete type.

Reply:
Agreed!